examples/js/effects/VREffect.js

```diff
@@ -141,17 +134,35 @@ THREE.VREffect = function ( renderer, onError ) {
 				eyeWidth = eyeParamsL.renderWidth;
 				eyeHeight = eyeParamsL.renderHeight;
 
+				if ( vrDisplay.getLayers ) {
+
+					var layers = vrDisplay.getLayers();
+					if (layers.length) {
+
+						leftBounds = layers[0].leftBounds || [ 0.0, 0.0, 0.5, 1.0 ];
```

Comment:
I believe the `VRLayer` object returned by `getLayers` can have `leftBounds` or `rightBounds` be `null` if either property was not set. The default value is only represented internally.

@toji can you confirm?